src/components/SourceDetail/ResourcesTable.js

```diff
@@ -146,19 +146,37 @@ const ResourcesTable = () => {
                       </Alert>
                     )}
                     {applicationsRows[app.id]?.length ? (
-                      <Table
-                        aria-label={intl.formatMessage({
-                          id: 'resourceTable.title',
-                          defaultMessage: 'Connected application resources',
-                        })}
-                        variant="compact"
-                        cells={columns}
-                        rows={applicationsRows[app.id]}
-                        className="pf-v5-u-mt-md"
-                      >
-                        <TableHeader />
-                        <TableBody />
-                      </Table>
+                      <React.Fragment>
```

Comment:
Data view?

Reply:
Not needed, this is static table wit no ordering and filtering.